src/views/AddTenant/addTenant.js

```diff
@@ -23,8 +23,10 @@ const validationSchema = Yup.object().shape({
     .min(7, "*Phone Number must have at least 7 characters")
     .max(20, "*Phone Number can't be longer than 20 characters")
     .required("*Phone Number is required"),
-  unitNum: Yup.number(),
-  occupants: Yup.number(),
+  unitNum: Yup.string()
+    .required("*Unit Number is required"),
+  occupants: Yup.number()
+    .required("*Number of Occupants is required"),
```

Comment:
These shouldn't be required. Sometimes properties can be detached homes which would not have a unitNum. 
We're not requiring these fields in the backend. However, perhaps occupants needs to be required in the backend? We could check with @htharker42 on this.

Reply:
I'm going to try to modify the JSON data so that empty values are submitted as `null` instead of `""`. Then, I can remove the required setting for these properties.